tests/unit/s2n_ktls_test_utils_test.c

```diff
@@ -157,8 +157,7 @@ int main(int argc, char **argv)
 
             uint8_t count = 5;
             size_t total_sent = 0;
-            struct iovec *send_msg_iov = NULL;
-            send_msg_iov = malloc(sizeof(*send_msg_iov) * count);
+            struct iovec send_msg_iov[sizeof(struct iovec) * 5] = { 0 };
```

Comment:
Defining a variable and not using it here seems like it makes it easy to change one value and forget to change the other. If you want, I think you can `#define count 5` so it can be used here. Alternatively, you could alloc a blob instead.

Reply:
Yea defenitely not ideal. Wouldnt the `#define` have to live at the top of the file.. that seems less than idea since it only used for this test but its prob still better than this. I'll use your suggestion